src/nc/input/le/LeParser.cpp

```diff
@@ -335,8 +334,21 @@ void LeParser::doParse(QIODevice *in, core::image::Image *image, const LogToken
             image->addRelocation(std::make_unique<core::image::Relocation>(
                     page_virt_addr + fh.srcoff,   // relocation virtual address
                     image->symbols()[dstobj - 1], // section alias as base
-                    width,                        // 4 byte relocations only
+                    width,
                     dst));                        // offset from section start
+
+            if (width == 4) {
+                uint32_t dst_virt = dst + image->sections()[dstobj - 1]->addr();
+                ByteOrder::convert(&dst_virt, width, ByteOrder::Current, bo);
```

Comment:
I guess, you can just write `po.convertTo(dst_virt)`.

Reply:
There's no method convertTo in that class, only convertFrom. Is adding convertTo better than my usage?

Reply:
I guess, it is absent, because it was never needed.
Could you add it?

Reply:
sure